test/core/end2end/invalid_call_argument_test.cc

```diff
@@ -137,8 +137,9 @@ static void cleanup_test() {
   grpc_completion_queue_shutdown(g_state.cq);
   while (grpc_completion_queue_next(g_state.cq,
                                     gpr_inf_future(GPR_CLOCK_REALTIME), nullptr)
-             .type != GRPC_QUEUE_SHUTDOWN)
+             .type != GRPC_QUEUE_SHUTDOWN) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done